tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -260,6 +281,33 @@ private string GetImage(string imageType, string version, string osVariant)
             return image;
         }
 
+        private static string GetRegistryName(string repo, string tag)
+        {
+            bool imageExistsInStaging = true;
+
+            // In the case of running this in a local development environment, there would likely be no image info file
+            // provided. In that case, the assumption is that the images exist in the staging location.
```

Comment:
> In that case, the assumption is that the images exist in the staging location.

This comment as well as the implementation doesn't seem correct.  In a local dev env, a staging location is not being used.  This would not be desirable behavior IMO.  If you build locally, you should end up with the mcr based tags.

Reply:
How did it work before?  Before my changes, it _never_ went to MCR.  So was the dev environment before incorrect before?

Reply:
It looks like `$"{Registry}/{RepoPrefix}"` resolves to `mcr.microsoft.com` in the dev flow because it the environment variables aren't being specified in which it retrieves the registry from the manifest.  So the proposal seem functional but not ideal in that it is not using the staging location in reality.

Reply:
Maybe instead of using "staging" terminology, I'll just use "custom".